src/stylesheets/components/_blank-slate.scss

```diff
@@ -0,0 +1,15 @@
+.blank-slate {
+  text-align: center;
+  padding-top: rem(250px);
+
+  &__emoji {
+    font-size: map-get($font-sizes, 'xxl');
```

Comment:
Should we use `font-size: rem(64px)` and `line-height: 100%` here
![Screen Shot 2563-11-11 at 12 21 18](https://user-images.githubusercontent.com/1772999/98772057-6f4c0480-2418-11eb-8b56-951d1293076f.png)